client/components/tinymce/plugins/wpcom/plugin.js

```diff
@@ -32,7 +32,7 @@ function wpcomPlugin( editor ) {
 
 		if ( event.content ) {
 			if ( event.content.indexOf( '<!--more' ) !== -1 ) {
-				title = __( 'Read more...' );
```

Comment:
That's a great point, and I've pulled 'Read more' out of i18n.js.

While we're on the topic, I'd like to check that using `i18n-calypso` makes sense from the perspective of someone with more investment in TinyMCE. `i18n-calypso` seems easier to me, but  I'm not sure what our reasons are/were for including `i18n.js`, and I wouldn't want to unilaterally reverse a thought out decision.